.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "kudo"]
+	path = kudo
+	url = https://github.com/kudobuilder/kudo.git
```

Comment:
Are we OK with master or would we want to pin a specific branch/tag?

Reply:
It's pinned to `v0.6.0` which has a SHA of `c74d22743d6ec37e24e89eaeb64803597187fe08`.